src/components/ThreadPreview.js

```diff
@@ -17,8 +17,8 @@ export const ThreadPreview = withRouter((props) => {
       <h4>{props.title}</h4>
       <button
         type="button"
-        className={"btn btn-outline-primary"}
-        onClick={() => {props.history.push('/thread/' + props.threadId)}}>
+        className="btn btn-outline-primary"
+        onClick={() => {props.history.push(`/thread/${  props.threadId}`)}}>
```

Comment:
And here:
```suggestion
        onClick={() => {props.history.push(`/thread/${props.threadId}`)}}>
```

Reply:
We will we using an internal <Button /> component for this in the future